crates/shared/src/order_validation.rs

```diff
@@ -140,7 +139,6 @@ pub enum ValidationError {
     InvalidQuote,
     /// Unable to compute quote because of a price estimation error.
     PriceForQuote(PriceEstimationError),
-    InsufficientFee,
```

Comment:
This probably needs an update in the OpenAPI specs as well.

Reply:
fixed